SEPADirectDebit/src/main/java/com/braintreepayments/api/SEPADirectDebitInternalTokenizeCallback.java

```diff
@@ -4,9 +4,9 @@
 
 /**
  * Callback for receiving result of
- * {@link SEPADirectDebitClient#tokenize(SEPADirectDebitPaymentAuthResult, SEPADirectDebitTokenizeCallback)}.
+ * {@link SEPADirectDebitClient#tokenize(SEPADirectDebitPaymentAuthResult, SEPADirectDebitInternalTokenizeCallback)}.
  */
-public interface SEPADirectDebitTokenizeCallback {
+public interface SEPADirectDebitInternalTokenizeCallback {
```

Comment:
What's the significance of the word `internal`?
We still expose this callback to the merchant, don't we?

Reply:
Ah whoops that should have been made package-private - updated!